koji_wrapper/base.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 
-"""
-Provides common functionality for classes in this module which
+"""Provides common functionality for classes in this module which
```

Comment:
https://www.python.org/dev/peps/pep-0257 is where I'm coming from and was following the python "hacking" module from openstack guidelines to find inconsistency, if we don't care or have preference for new line then I'll rotate that change out of this changeset.

Reply:
fwiw, I don't use the new-line style in git_wrapper, however using single quote over double is not consistent.

Reply:
Ok, thank you for the reference, I just went and took a look, as I like consistency as well.  In the milt-line section[1], I see the following in the first paragraph:
```
The summary line may be on the same line as the opening quotes or on the next line. 
```
If nobody agrees with me about keeping it on a separate line, fine, otherwise, I would prefer we keep the line break.

[1] https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Consistent is all I ask.

Reply:
same here and I'll tweak my config to ignore that entry then